src/EndpointBuilder.ts

```diff
@@ -1,20 +1,18 @@
-// @flow
-
 import Endpoint from './Endpoint'
-import type { MapResponseType } from './MapResponseType'
-import type { MapParamsToUrlType } from './MapParamsToUrlType'
-import type { MapParamsToBodyType } from './MapParamsToBody'
+import { MapResponseType } from './MapResponseType'
+import { MapParamsToUrlType } from './MapParamsToUrlType'
+import { MapParamsToBodyType } from './MapParamsToBody'
 
 /**
  * Helper class to build a {@link Endpoint}
  */
 class EndpointBuilder<P, T> {
   name: string
-  paramsToBodyMapper: ?MapParamsToBodyType<P>
-  paramsToUrlMapper: MapParamsToUrlType<P>
-  mapper: MapResponseType<P, T>
-  responseOverride: ?T
-  errorOverride: ?Error
+  paramsToBodyMapper: MapParamsToBodyType<P> | null | undefined
+  paramsToUrlMapper: MapParamsToUrlType<P> | null | undefined
+  mapper: MapResponseType<P, T> | null | undefined
+  responseOverride: T | null | undefined
+  errorOverride: Error | null | undefined
```

Comment:
I know I am not assigned to this PR, but as I've already worked with TS I could not hold back mentioning that the proper way to replace optionals would probably be
```suggestion
  errorOverride?: Error
```